tests/rkt_api_service_test.go

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// +build coreos
```

Comment:
I actually do like the explicity of whitelisting. it's also the safer approach, e.g. you do not want to run anything using fly on your development machine accidentally.